### PR TITLE
Fixed validation messages not showing on failed submit

### PR DESCRIPTION
**Pull request by sarj21**
Semantic UI doesn't show the error messages unless its a form and it has the "ui error message" div in the form
Screenshot: image
This wasn't showing up before and made long forms (like confirmation) confusing

### CODE REVIEW

File: app/client/views/application/application.html [hunk 1]
```diff
@@ -5,121 +5,123 @@
...
     <div class="column">
-      <div class="ui form"
+      <form class="ui form"
             ng-class="{'loading': loading}">
 
         <div class="divided title">Basic Information</div>
 
         <fieldset ng-disabled="regIsClosed">
 
-        <div class="field">
```
Comment (sarj21) [inline app/client/views/application/application.html:15]: this was just one level of indentation off so i pushed it in one, but the diff is insane...

**Comment by sarj21** [comment]
Nothing to see here, this isn't the worst branch of all time

**Comment by jamestaylr** [comment]
Is this a change we would want to merge into upstream?

**Comment by sarj21** [comment]
on a clean branch sure. I'm not sure if this was oversight on their part or if its just something didn't want. I can submit a PR though and they can decide

File: app/client/views/application/application.html (hunk 1 above)
Comment (mecolmg) [reply app/client/views/application/application.html:15]: AHHHHHH

File: app/client/views/application/application.html [hunk 2]
```diff
@@ -5,121 +5,123 @@
 
   <div class="ui stackable page grid">
     <div class="column">
-      <div class="ui form"
+      <form class="ui form"
```
Comment (mecolmg) [inline app/client/views/application/application.html:8]: This is one of those things that makes web dev infuriating

**Review by mecolmg — APPROVED** [review]
Lgtm

**Pull request merged**